src/components/ens-registration/SearchResult/SearchResultBasicIndicator.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Does this need to be a component? 😅 I guess it feels a bit awkward having mutually exclusive props (`registrationDate` and `feesCost` are unrelated to each other). I feel like it would be simpler just to have the content within the RegisterEnsSheet.

Otherwise, the styling and all that looks good!

Reply:
haha i suppose you are right